pkg/madmin/API.md

```diff
@@ -121,15 +121,106 @@ If successful restarts the running minio service, for distributed setup restarts
 
  __Example__
 
- 
+
  ```go
 
 
 	st, err := madmClnt.ServiceRestart()
 	if err != nil {
 		log.Fatalln(err)
 	}
-	log.Printf("Succes")
+	log.Printf("Success")
 
  ```
+<a name="ListLocks"></a>
+### ListLocks(bucket, prefix string, olderThan time.Duration) ([]VolumeLockInfo, error)
+If successful returns information on the list of locks held on ``bucket`` matching ``prefix`` older than ``olderThan`` seconds.
+
+__Example__
+
+``` go
+    volLocks, err := madmClnt.ListLocks("mybucket", "myprefix", 30 * time.Second)
+    if err != nil {
+        log.Fatalln(err)
+    }
+    log.Println("List of locks: ", volLocks)
+
+```
+
+<a name="ClearLocks"></a>
+### ClearLocks(bucket, prefix string, olderThan time.Duration) ([]VolumeLockInfo, error)
+If successful returns information on the list of locks cleared on ``bucket`` matching ``prefix`` older than ``olderThan`` seconds.
+
+__Example__
 
+``` go
+    volLocks, err := madmClnt.ClearLocks("mybucket", "myprefix", 30 * time.Second)
+    if err != nil {
+        log.Fatalln(err)
+    }
+    log.Println("List of locks cleared: ", volLocks)
+
+```
+
+<a name="ListObjectsHeal"></a>
+### ListObjectsHeal(bucket, prefix string, recursive bool, doneCh <-chan struct{}) (<-chan ObjectInfo, error)
+If successful returns information on the list of objects that need healing in ``bucket`` matching ``prefix``.
+
+__Example__
+
+``` go
+    // Create a done channel to control 'ListObjectsHeal' go routine.
+    doneCh := make(chan struct{})
+
+    // Indicate to our routine to exit cleanly upon return.
+    defer close(doneCh)
+
+    // Set true if recursive listing is needed.
+    isRecursive := true
+    // List objects that need healing for a given bucket and
+    // prefix.
+    healObjectCh, err := madmClnt.ListObjectsHeal("mybucket", "myprefix", isRecursive, doneCh)
```

Comment:
This example can talk about ListObjectsHeal and calling -HealObject() on it. With some conditional changes based on HealInfo.